From 5af98ca8135ac411364b16720d795224a9b4a178 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Sat, 30 Jul 2016 15:15:14 +0200
Subject: [PATCH] Makefile: allow to pass additional LIBS

We need to be able to pass extra LIBS when our toolchain lacks NLS
support, this way we can build libintl and link to it.  A good example
is uClibc with locale support disabled.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 src/Makefile     | 2 +-
 src/gui/Makefile | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index b50586b..acbdbfa 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -30,7 +30,7 @@ ifneq ($(shell $(LD) --help 2| grep -- --as-needed), )
 	LDFLAGS+= -Wl,--as-needed
 endif
 LDSTATIC=-static
-LIBS=-llshw -lresolv
+LIBS+=-llshw -lresolv
 ifeq ($(SQLITE), 1)
 	LIBS+= $(shell pkg-config --libs sqlite3)
 endif
diff --git a/src/gui/Makefile b/src/gui/Makefile
index 332ce57..7f72e3f 100644
--- a/src/gui/Makefile
+++ b/src/gui/Makefile
@@ -11,7 +11,7 @@ INCLUDES=-I../core $(GTKINCLUDES)
 CXXFLAGS=-g -Wall $(INCLUDES) $(DEFINES) $(RPM_OPT_FLAGS)
 CFLAGS=$(CXXFLAGS) $(DEFINES)
 GTKLIBS=$(shell pkg-config gtk+-2.0 gmodule-2.0 --libs)
-LIBS=-L../core -llshw -lresolv $(GTKLIBS)
+LIBS+=-L../core -llshw -lresolv $(GTKLIBS)
 LDFLAGS=
 ifneq ($(shell $(LD) --help 2| grep -- --as-needed), )
 	LDFLAGS+= -Wl,--as-needed
-- 
2.7.4

